packages/expo-firebase-analytics/package.json

```diff
@@ -35,10 +35,8 @@
   "devDependencies": {
     "expo-module-scripts": "~1.2.0"
   },
-  "unimodulePeerDependencies": {
-    "@unimodules/core": "*"
-  },
   "peerDependencies": {
+    "@unimodules/core": "*",
```

Comment:
the reason we don't put this as a regular peer dep is that if you install `react-native-unimodules` then `@unimodules/core` is a dep of that, and peer deps can only be satisfied by direct dependencies so even though the code will work the user will get a warning on install